src/main/java/be/syntra/devshop/DevshopFront/services/utils/CategoryMapper.java

```diff
@@ -14,4 +14,10 @@
         categories.forEach(category -> categoryNames.add(category.getName()));
         return categoryNames;
     }
+
+    public List<Category> mapToCategories(List<String> categoryNames) {
```

Comment:
this method is only used in an unused method. Remove both

Reply:
removed